app/views/group_assignments/_group_assignment_form_options.html.erb

```diff
@@ -19,7 +19,10 @@
 
 <dl class="form">
   <dt><label>Add your starter code from GitHub <small>(optional)</small> </label></dt>
-  <dd><input class="textfield" type="text" name="repo_name" value="<%= group_assignment.full_name %>" placeholder="<%= group_assignment.full_name || 'e.g. education/classroom'%>"</dd>
+  <dd><input id="autocomplete-textfield" class="textfield" type="text" name="repo_name" value="<%= group_assignment.full_name %>" placeholder="<%= group_assignment.full_name || 'e.g. education/classroom'%>"></dd>
+
+  <!-- autocomplete suggestions -->
```

Comment:
I'm not really a big fan of comments in the markup could you also remove this comment?